### ISSUE TITLE
Output from tasks is not streaming

### ISSUE THREAD

**Issue by sheerun** [go-task/task#190]
I have long-running task that outputs some logs in the console. Unfortunately when it's run through go-task, the output is displayed only when task finishes, which can take long time.

Is it possible to receive streaming output from currently executed task? i.e. how to make go-task output line of logs immediately after running task is outputting it

**Comment by andreynering** [comment]
Hi @sheerun,

Task has a configuration to choose how it redirects output from commands: https://taskfile.dev/#/usage?id=output-syntax

Which one are you using? (Default is `interleaved`).

The behavior you're describing is the expected for `group`. But if you're `interleaved` or `prefixed` then we have a bug, and I'll need to know form you how to reproduce it.

**Comment by sheerun** [comment]
I'm using the defaults. Here is my Taskfile:

```
version: '2'

tasks:
  default:
    cmds:
      - ruby -e "10.times { |i| puts i; sleep 1 }"
```

I'd expect that it has streaming output

**Comment by andreynering** [comment]
Indeed, there seems to be a bug there.

I haven't noticed that for other commands. Maybe it's specific to Ruby.

**Comment by andreynering** [comment]
Hi @sheerun,

This should be fixed on master on after #200. Please, let me know if you're still experiencing problems.
